packages/block-library/src/site-logo/edit.js

```diff
@@ -461,17 +489,48 @@ export default function LogoEdit( {
 		createErrorNotice( message, { type: 'snackbar' } );
 	};
 
+	const onFilesDrop = ( filesList ) => {
+		mediaUpload( {
+			allowedTypes: [ 'image' ],
+			filesList,
+			onFileChange( [ image ] ) {
+				if ( isBlobURL( image?.url ) ) {
+					return;
+				}
+				onInitialSelectLogo( image );
+			},
+			onError: onUploadError,
+		} );
+	};
+
+	const InspectorLogoPreview = ( { itemGroupProps } ) => {
```

Comment:
Can we define this component outside of the Edit component? So we don't have to re-create it on every rerender. I believe that's the recommended practice in React.